### ISSUE TITLE
NVDA Dialog focusing issues

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2925]
**Reported by Q on 2013-01-11 21:55**
When I invoke a dialog such as NVDA find, or list elements in Firefox, sometimes after approximately one second the dialog loses focus, and focus is returned to the firefox window. Reinvoking the dialog makes a second dialog appear, and once that second dialog is dismissed, the original dialog is once-again focused. It is almost as if the NVDA windows are trying but failing to steal focus, but I will not speculate as I have no idea what's going on here.
Ah, further information: The find and other dialogs appear to show up in alt-tab order when they are launched, and subsequently automatically unfocused.

**Comment by ehollig** [comment]
@ctoth, were you the original Q who made this ticket? If so, could you provide more information to reproduce this if it is still happening?

**Comment by ehollig** [comment]
Closing as invalid, as we currently have insufficient information to reproduce the issue.
The issue with multiple find dialogs is covered in #4228
